pycovid/pycovid.py

```diff
@@ -112,7 +123,7 @@ def getIntervalData(df, interval='30D'):
 
 def plot_countries(df=None, grouped_data=False, metric="confirmed"):
     
-    if df is None:
+    if df == None:
```

Comment:
Actually, testing for "None" is better via "is". "None" is a singleton in Python, while "==" can do all kind of surprising things (eg on numpy arrays)

Reply:
https://github.com/sudharshan-ashok/pycovid/pull/7